GitUI/UserControls/RevisionGrid/RevisionGridMenuCommands.cs

```diff
@@ -421,5 +433,35 @@ public void GotoCommitExecute()
                 }
             }
         }
+
+        public void SwitchBetweenArtificialCommits()
```

Comment:
This method switches betwen 2 artificial commits and the current HEAD, which isn't artificial. So the name is a little misleading.